src/components/layout/nav-careers.js

```diff
@@ -47,43 +72,53 @@ export const NavCareers = () => {
                 <StyledNav>
                     <Wrapper>
                         <CareerNavLeft>
-                            <CareerLogo to={'/'} aria-label={localize('Home')}>
+                            <CareerLogo to={'/'} aria-label={'Home'}>
                                 <QueryImage
                                     data={data['deriv']}
-                                    alt={localize('Deriv')}
+                                    alt={'Deriv'}
                                     width="16.4rem"
                                     height="auto"
                                 />
                             </CareerLogo>
-                            <StyledLink
+                            <CareerLink
                                 activeClassName="active"
                                 to="/careers"
-                                aria-label={localize('Careers home')}
+                                aria-label={'Careers'}
                                 partiallyActive={true}
                             >
                                 HOME
-                            </StyledLink>
-                            <StyledLink
+                            </CareerLink>
+                            <CareerLink
                                 activeClassName="active"
                                 to="/careers/locations/"
-                                aria-label={localize('Locations')}
+                                aria-label={'Locations'}
                                 partiallyActive={true}
                             >
                                 LOCATIONS
-                            </StyledLink>
+                            </CareerLink>
+                            <CareerLink
+                                activeClassName="active"
+                                to="/"
```

Comment:
Is this the same as passing an empty string to `to` whenever there is another prop of `is_example_link`? I've done so in the past.

Reply:
its not the same, its adding `/` in the end of the url, only for consistency purposes